chapters/13.xml

```diff
@@ -3253,11 +3242,11 @@
         <natlang><quote>How's that again, John?</quote> said George.</natlang>
       </interlinear-gloss>
       <interlinear-gloss>
-        <jbo>.i sei la pam. cusku se'u</jbo>
-        <gloss>- [Comment] that-named Pam says, [end-comment]</gloss>
+        <jbo>.i sei la .pam. cu cusku se'u</jbo>
+        <gloss>- [Comment] that-named Pam - says, [end-comment]</gloss>
       </interlinear-gloss>
       <interlinear-gloss>
-        <jbo>… ju'i .djordj. .e'unai le kabri ba zi farlu</jbo>
+        <jbo>… ju'i .djordj. o'i le kabri ba zi farlu</jbo>
```

Comment:
.ei pei jdice lo du'u ma kau sa'e padle'u javni .i zoi ba« .djordj. .o'i »ba zo'u no'e vajni mi .i zu'u nai .au nai lo simsa be zoi ba« by. cy. .y. .abu cy. .y'y.bu »ba noi so'u da pilno cu stali